tracer/src/Datadog.Trace/Span.cs

```diff
@@ -494,6 +497,21 @@ internal void SetDuration(TimeSpan duration)
             Duration = duration;
         }
 
+        internal IReadOnlyDictionary<string, object> GetMetaStructValues()
+        {
+            return _metaStruct.IsValueCreated ? _metaStruct.Value : null;
+        }
+
+        internal bool AddMetaStructValue(string key, object value)
+        {
+            if (!string.IsNullOrEmpty(key))
```

Comment:
We already handle string tags and numeric tags using `TagsList.cs`. Since `meta_struct` adds a third type of span tag, we should consider adding that to `TagsList` and adding these `Get()` and `Set()` APIs there.

| Tag Type  | MsgPack Field Name |
|-----------|--------------------|
| string    | `meta`             |
| numeric   | `metrics`          |
| raw bytes | `meta_struct`      |

Reply:
💯

Reply:
I have refactored the code to do it like the others. Thanks!